wordpress-mobile/WordPress-Android
# Make reader blog section wrap content in case of large text

Fixes #14589 

This fix replaces the previous fixed height of the title and url with min height and wrap content. The result with the largest text doesn't look perfect (the site icon is not centered with the text) but it stops the URL crop and I think this is quite edge casey anyway. 

To test:
- Increase font size to largest
- Go to reader
- Notice the URL is no longer cropped
- Decrease font size to normal
- Check the UI looks the same as before


<img width="490" alt="Screenshot 2021-05-04 at 11 42 48" src="https://user-images.githubusercontent.com/1079756/116986494-22adfb80-acce-11eb-8a6a-a1d717a58946.png">


## Regression Notes
1. Potential unintended areas of impact
- I can't see any

2. What I did to test those areas of impact (or what existing automated tests I relied on)
- n/a

3. What automated tests I added (or what prevented me from doing so)
- n/a


PR submission checklist:
